src/test/java/com/cag/cagbackendapi/controllers/UserControllerTest.kt

```diff
@@ -46,9 +46,9 @@ class UserControllerTest {
     }
 
     @Test
-    fun registerUser_missingFirstNameAndLastNameAndEmail_400BadRequest() {
+    fun registerUser_missingFirstNameAndLastNameAndEmailAndPassword_400BadRequest() {
         val testAuthKey = "testAuthKey"
-        val requestUser = UserRegistrationDto(first_name = null, last_name = null, email = null, agreed_18 = true)
+        val requestUser = UserRegistrationDto(first_name = null, last_name = null, email = null, pass = null, agreed_18 = true)
 
         val badRequestException = BadRequestException(DetailedErrorMessages.FIRST_NAME_REQUIRED + DetailedErrorMessages.LAST_NAME_REQUIRED + DetailedErrorMessages.EMAIL_REQUIRED, null)
```

Comment:
Since password is `null`, shouldn't we also expect the `DetailedErrorMessages.PASSWORD_REQUIRED` message as well?

Reply:
Great catch! Updated